Game/MyProject/Game/Helper/MeshManager.hpp

```diff
@@ -0,0 +1,27 @@
+//-------------------------------------------------------------------------------
+//-- MeshManager.h --------------------------------------------------------------
+//-------------------------------------------------------------------------------
+#pragma once
+//-------------------------------------------------------------------------------
+//-- Dependencies ---------------------------------------------------------------
+//-------------------------------------------------------------------------------
+#include "Mesh.hpp"
+#include "HelperManager.hpp"
+#include <stdio.h>
+#include <map>
+//-------------------------------------------------------------------------------
+//-- Classes --------------------------------------------------------------------
+//-------------------------------------------------------------------------------
+class MeshManager
```

Comment:
what's the purpose of this class?

Reply:
I need only a mesh for a sphere,box,quad and many instances. So this class keeps track of it. I create the mesh only once and use it.

Reply:
sure, or you could just keep a static reference in the get method and skip the whole get by name and additional class. anyway, let's not waste time redoing this, it's fine